### DISCUSSION TITLE
LAMBDA_RUNTIME occurs only while using getServerSideProps() function in a page.

### DISCUSSION THREAD

**Discussion by isas5675** [vercel/community#2327] [Help]
### Problem Description

We created a project using NextJS and deployed it on Vercel.

If we use `getServerSideProps()` in a file and deploy it to Vercel, we get the following error:

`[ERROR] [1683285000358] LAMBDA_RUNTIME Failed to post handler success response. Http response code: 413.`

The `getServerSideProps()` function is a totally empty function and we are only returning an empty property in the props object (which is totally valid syntax).

If we remove `getServerSideProps()` then we don't get the error at all. We don't get that error even if we use `getStaticProps()`, or simply use client side rendering. 

We are simply not able to understand why are we getting that error. Can anyone please help us with this? 

Here's the code:

-------------------------------------------------------------------------------------

```
// inside index.js file
import React from "react";

export async function getServerSideProps(context) {
  return {
    props: {
      foo: "bar",
    }
  }
}

export default function Home(props) {
  return (
    <>
      <h1>Hello World</h1>
    </>
  )
}
```

**Comment by amyegan** [comment]
If you're getting a 413 error with code [FUNCTION_PAYLOAD_TOO_LARGE](https://vercel.com/docs/errors/application-error-codes#function_payload_too_large) using the above code, that certainly does seem strange. I suspect there is something elsewhere in the code or configuration affecting this. Do you have a public repo or minimal reproduction that we can use for debugging?

**Comment by isas5675** [comment]
Restarting the server solved the issue. Thank you for your time.
